### PR TITLE
The supermatter now has a ghost-visible damage display

### PR DESCRIPTION
**Pull request by ChangelingRain**
I don't know if anyone asked for this but ho!

### CODE REVIEW

**Comment by PKPenguin321** [comment]
Put megafauna health percentage over them for ghosts as well

**Comment by ChangelingRain** [comment]
megafauna have the tiny corner health display though! ghosts don't NEED that!

**Comment by PKPenguin321** [comment]
oh i never thought of that

File: code/game/objects/effects/countdown.dm [hunk 1]
```diff
@@ -125,6 +125,17 @@
...
+/obj/effect/countdown/supermatter
+	name = "supermatter damage"
+	text_size = 1
+	color = "#ED84F4"
+
+/obj/effect/countdown/supermatter/get_value()
+	var/obj/machinery/power/supermatter_shard/S = attached_to
+	if(!istype(S))
+		return
+	return "<div align='center' valign='middle' style='position:relative; top:0px; left:6px'>[round((S.damage / S.explosion_point) * 100)]</div>"
```
Comment (coiax) [inline code/game/objects/effects/countdown.dm]: why can't you make the rest of the countdowns pretty Joan

Comment (ChangelingRain) [inline code/game/objects/effects/countdown.dm]: This is not the pretty fancy countdown pr, silly

Comment (coiax) [inline code/game/objects/effects/countdown.dm]: but you're so good at it :'(

**Comment by MrStonedOne** [comment]
feature

**Comment by coiax** [comment]
It's QoL

**Comment by coiax** [comment]
isn't #22531 test merged right now?

**Comment by ChangelingRain** [comment]
Are you just trying to use my pr as leverage to reopen your pr? Because *get the fuck out of my pr.*

**Comment by MrStonedOne** [comment]
>isn't #22531 test merged right now?

Ya, and i'm having to suspend all updates to keep it test merged.

Thats not an argument.

**Comment by coiax** [comment]
Someone shout at @optimumtact to reopen #22531

**Comment by optimumtact** [comment]
Blah blah blah I'm a big whiny baby

**Pull request merged**